frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/initial-state/cloud-init-tab-initial-state.ts

```diff
@@ -1,11 +1,14 @@
 import { CloudInitField } from '../../types';
+import { StepState } from './types';
 
-export const getCloudInitInitialState = () => ({
+export const getCloudInitInitialState: () => StepState = () => ({
```

Comment:
Can we introduce new type for these initalState functions? It will improve readability of the code.

Reply:
fixed, + added additional types